package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   summary: 'Accounts Templates styled for Semantic UI.',
-  version: '1.11.1',
+  version: '1.12.0',
```

Comment:
Could you please revert back to 1.11.1?
I have some scripts that will automatically bump at the next publish operation...